src/omnisharp/omnisharpDownloader.ts

```diff
@@ -66,16 +70,83 @@ export class OmnisharpDownloader {
         return false;
     }
 
-    public async GetLatestVersion(serverUrl: string, latestVersionFileServerPath: string): Promise<string> {
-        const description = 'Latest OmniSharp Version Information';
-        const url = `${serverUrl}/${latestVersionFileServerPath}`;
+    public async GetLatestVersion(serverUrl: string): Promise<string> {
+        const url = `${serverUrl}/releases?per_page=1`;
         try {
             this.eventStream.post(new LatestBuildDownloadStart());
-            const versionBuffer = await DownloadFile(description, this.eventStream, this.networkSettingsProvider, url);
-            return versionBuffer.toString('utf8');
+            const releases = await this.fetchLatestReleases(url);
+            const latestRelease = releases[0];
+            return latestRelease.tag_name.replace('v', '');
         } catch (error) {
             this.eventStream.post(new InstallationFailure('getLatestVersionInfoFile', error));
             throw error;
         }
     }
+
+    public async fetchLatestReleases(urlString: string): Promise<any> {
```

Comment:
This is mostly the existing `downloadFile` with cancellation and progress reporting removed.